src/cms/models/pages/page_translation.py

```diff
@@ -119,6 +119,20 @@ def combined_text(self):
             return attached_text + self.text
         return self.text + attached_text
 
+    @property
+    def combined_last_updated(self):
+        """
+        This function combines the last_updated date from this PageTranslation and from a mirrored page.
+        If this translation has no content, then the date from the mirrored translation will be used. In
+        other cases, the date from this translation will be used.
+
+        :return: The last_upated date of this or the mirrored page translation
+        :rtype: ~django.db.models.DateTimeField
```

Comment:
```suggestion
        :rtype: ~datetime.datetime
```
The property doesn't return the field, it returns the content of the field. Thus, the return type is `datetime` directly.

Reply:
I need to get used to this concept :grimacing: